lib/lookerUtils.ts

```diff
@@ -0,0 +1,35 @@
+import { getProposedEndDate } from "./experimentUtils";
+
+// XXX getExperimentLookerDashboardDates and getLookerSubmissionTimestampDateFilter
+// were moved to this file rather than looker.ts due to "Module not found" errors.
+// These errors were likely made because looker.ts has functions that use node
+// modules that should only exist in server components. But these functions are 
+// passed through other functions that eventually go through columns.tsx which has
+// "use client". We should look fo a better architectural solution regarding 
+// the situation with these errors.
```

Comment:
Probably good to say why here.  Something along the lines of doing this both for separation of concerns/code maintainability, but also so that we pre-render more of the markup on the server for performance reasons.